server/src/main/java/org/apache/iotdb/db/metadata/mtree/MTree.java

```diff
@@ -1783,6 +1783,10 @@ protected boolean processFullMatchedMNode(IMNode node, int idx, int level)
                 return true;
               }
 
+              if (node.isMeasurement()) {
+                return false;
```

Comment:
Excellent correction! While `return false` should have fixed this issue, but `return true` might be a clearer statement that the `traverser` should not dive into this branch any more.

Reply:
You are righ

Reply:
fixed~